index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const porject_env = require('dotenv').config();
+const { Pool } = require('pg')
+
+const app = express();
+
+const pool = new Pool({
+  user: process.env.DB_USERNAME,
```

Comment:
setup mailingServer const in constants.js
mailingServer will hold mailchimp for testing/local env and maingun for production env.